### DISCUSSION TITLE
Should spin template use tags

### DISCUSSION THREAD

**Discussion by schnetzi** [serversideup/spin#141] [Q&A] [answered]
The question is if spin templates should get tagged when the code is updated or if it should anyway always use the latest commit?

**Comment by jaydrogers** [comment, marked as answer]
What we do for now is just use the latest commit on the branch. We have releases, but they are basically just notes to be emailed to people (example https://github.com/serversideup/spin-template-laravel-basic).

If you want to do testing, then you can do things like this when we did extensive testing with Laravel Pro:

![image](https://github.com/user-attachments/assets/7c6afc3f-d8cb-4431-a72e-ade6b138bc03)
